file/file_prefetch_buffer.cc

```diff
@@ -660,8 +660,8 @@ bool FilePrefetchBuffer::TryReadFromCacheUntracked(
             return false;
           }
         }
-        ReadAheadSizeTuning(offset, n);
-        s = Prefetch(opts, reader, offset, n + readahead_size_);
+        size_t current_readahead_size = ReadAheadSizeTuning(offset, n);
+        s = Prefetch(opts, reader, offset, n + current_readahead_size);
```

Comment:
Prefetch function uses passed arg (n + current_readahead_size) instead of readahead_size_